coreprofile/10/_index.md

```diff
@@ -34,7 +34,26 @@ The goals for this release were:
 
 ## Release Review
 
-TBD
+The Specification Committee Ballot concluded successfully on 2022-08-23 with the following results.
```

Comment:
Not a big thing, but according to the ballot email, the ballot ended on August 15. I usually use that date even if it is announced on the mailing list later.

Reply:
okay, thanks. updated.